cmd/wallet/wallet.toml

```diff
@@ -1,4 +1,4 @@
-KSLocation = "/tmp/meson-wallet/keystore"
+KSLocation = "/Users/liaoyuchen/Developer/Meson-wallet-demo/keystore"
```

Comment:
Change the location of this since this is unique to your own machine

Reply:
Changed, thanks!